step/step.go

```diff
@@ -184,6 +192,24 @@ func (s XcodebuildTester) ExportOutputs(result Result) error {
 	return nil
 }
 
+func (s XcodebuildTester) getSimulatorForDestination(destinationSpecifier string) (destination.Device, error) {
+	simulatorDestination, err := destination.NewSimulator(destinationSpecifier)
+	if err != nil {
+		return destination.Device{}, fmt.Errorf("invalid destination specifier (%s): %w", destinationSpecifier, err)
+	}
+
+	s.logger.Println()
+	device, err := s.deviceFinder.FindDevice(*simulatorDestination)
+	if err != nil {
+		return destination.Device{}, fmt.Errorf("simulator UDID lookup failed: %w", err)
+	}
+
+	s.logger.Infof("Simulator info")
```

Comment:
Shouldn't we print this in the `ProcessConfig` func?

Reply:
Moved logging to ProcessConfig.